api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -208,7 +244,13 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	 */
 	public void purgeConceptDatatype(ConceptDatatype cd) throws DAOException;
 	
+=======
+>>>>>>> bae085a53... TRUNK-1824: Removing  ConceptDAO.saveConceptDatatype() , ConceptDAO.purgeConceptDatatype() , HibernateConceptDAO.saveConceptDatatype() and  HibernateConceptDAO.purgeConceptDatatype()
 	/**
+=======
+>>>>>>> f57b7f645... TRUNK-1824 : Refactoring and adding more javadocs to ConceptDAO
+=======
+>>>>>>> 34bf244e7583661b01a7a249092ea16f43e7c6c6
```

Comment:
Are you sure you haven't forgotten to delete something? ;)

Reply:
@sacull  very true i had some conflicts which am resolving locally.